kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/internal/FilterHandler.java

```diff
@@ -298,16 +298,11 @@ private <F extends FilterResult> F handleFilteringException(Throwable t, Decoded
 
     private <F extends FilterResult> CompletableFuture<F> handleDeferredStage(DecodedFrame<?, ?> decodedFrame, CompletableFuture<F> future) {
```

Comment:
`decodedFrame` is currently redundant in this PR as it was only passed in to work out if it was a request or response for logging purposes. 

I'm conflicted about the removal of the logging.

Reply:
AS discussed in slack, we've standardised on the logging level and will used the same message as the exception for logging. I've kept the usage of `decodedFrame` as I think it provides a better message